src/dkg/pv.rs

```diff
@@ -50,17 +51,18 @@ where
             participants: vec![],
             vss: BTreeMap::new(),
             domain,
-            state: DKGState::<E>::Init {
-                announce_messages: vec![],
-            },
+            state: DKGState::Init,
             me: 0, // TODO: invalid value
             //final_state: None,
             local_shares: vec![],
+            // TODO: Validators don't announce themselves through DKG
+            // TODO: Instead, we read stakes from storage
+            announce_messages: vec![],
```

Comment:
Do we even need to store these anymore? I don't think so. I think we can pass this information into finalize_announcement below as a parameter. All we really need stored here is the `participants`